lib/matplotlib/testing/compare.py

```diff
@@ -284,7 +294,12 @@ def convert(filename, cache):
                 return str(newpath)
 
         _log.debug("For %s: converting to png.", filename)
-        converter[path.suffix[1:]](path, newpath)
+        convert = converter[path.suffix[1:]]
+        if path.suffix == ".svg":
```

Comment:
maybe a comment why we patch the converter.

Reply:
done